dashboard/src/components/Catalog/Catalog.tsx

```diff
@@ -235,7 +237,10 @@ export default function Catalog() {
     repos
       .filter(r => !r.namespaceScoped || r.packageRepoRef?.context?.namespace === namespace)
       .map(r => r.name),
-  );
+  ).sort();
+  const allPlugins = uniq(
+    configuredPlugins.filter(p => p.name.includes(".packages")).map(p => getPluginName(p) || ""),
```

Comment:
Is this `.packages` plugin name a convention?
Perhaps in the future the getConfiguredPlugins API could return a specific field determining the type of plugin.

Reply:
Yes, currently all the plugins are using this `packaging` prefix, so it's safe to use it here (but it's an ugly workaround, though).
As we are adding new plugins (thinking now of the RBAC one we discussed), we may want to add a `type` field, good point; we should do it.